### PR TITLE
Map for mobile

### PR DESCRIPTION
**Pull request by Yuriks1**
Divided map into three parts and created three separate components for each pair of continents.

### CODE REVIEW

**Review by DarkendHall — CHANGES_REQUESTED** [review]

File: frontend/src/components/ContinentMapMobileLeft.vue [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        .on('mouseover', mouseOver)
+        .on('mouseleave', mouseLeave)
+        .on('click', mouseClick);
+  });
+});
+</script>
+
+<template>
+  <div class="header_under__title">
+    <h1>N.America / S. America</h1>
```
Comment (ToniKaru) [inline frontend/src/components/ContinentMapMobileLeft.vue:95]:
```suggestion
    <h1>The Americas</h1>
```

File: frontend/package.json [hunk 2]
```diff
@@ -14,8 +14,11 @@
         "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path ../.gitignore"
     },
     "dependencies": {
+        "@mdi/font": "^7.0.96",
+        "@mdi/js": "^7.0.96",
```
[the remark this answers is not in the document]
Comment (Yuriks1) [reply frontend/package.json]: No, these can be removed.

File: frontend/src/views/HomeView.vue [hunk 3]
```diff
@@ -3,12 +3,13 @@ import GeoButton from '@/components/GeoButton.vue';
...
 import router from '@/router';
 import MainMenuContainer from '@/components/MainMenuContainer.vue';
+import { isMobileOnly	 } from 'mobile-device-detect';
 </script>
 
 <template>
         <MainMenuContainer>
             <div class="wrapper">
-                <GeoButton color="green" size="large" @click="router.push('/quiz')"> Play now</GeoButton>
+                <GeoButton color="green" size="large" @click="router.push(isMobileOnly?'/quizMob':'/quiz')"> Play now</GeoButton>
```
[the remark this answers is not in the document]
Comment (Yuriks1) [reply frontend/src/views/HomeView.vue]: Worked for me and Toni, needs to refresh for it to function.

File: frontend/src/components/ContinentMapMobile.vue [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+  margin-top: 116px;
+  gap: calc(var(--gap) * 2);
+}
+
+.header_under__map {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+</style>
```
[the remark this answers is not in the document]
Comment (Yuriks1) [reply frontend/src/components/ContinentMapMobile.vue:150]: Fixed indents.

File: frontend/src/components/ContinentMapMobileRight.vue [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  justify-content: center;
+  margin-top: 116px;
+  gap: calc(var(--gap) * 2);
+}
+.header_under__map {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+</style>
```
[the remark this answers is not in the document]
Comment (Yuriks1) [reply frontend/src/components/ContinentMapMobileRight.vue:132]: Fixed indents.

File: frontend/src/components/ContinentMapMobileLeft.vue [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+  justify-content: center;
+  margin-top: 116px;
+  gap: calc(var(--gap) * 2);
+}
+.header_under__map {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+</style>
```
[the remark this answers is not in the document]
Comment (Yuriks1) [reply frontend/src/components/ContinentMapMobileLeft.vue:132]: Fixed indents.

**Review by DarkendHall — APPROVED** [review]

**Review by ToniKaru — APPROVED** [review]
Everything works like butter!

**Pull request merged**